### ISSUE TITLE
Modify offset position

### ISSUE THREAD

**Issue by G-J** [FezVrasta/popper.js#486]
### CodePen demo

https://codepen.io/gjk/pen/OOXQZe

### Steps to reproduce the problem

1.  initialize a new Popper
2.  Set the modifiers offset to 0%

### What is the expected behavior?

I expect the popper to be positioned at the absolute bottom left of the reference.

### What went wrong?

The popper is gets the offset left value of the reference container. Placing it x px along the axis. I would have expected the positioning in the inline transform attribute to place it at 'left 0'

### Any other comments?

Can this also be done straight from Bootstrap dropdown initialization?

**Comment by Johann-S** [comment]
Yep you can do that on Bootstrap dropdown by using `data-offset` see : https://getbootstrap.com/docs/4.0/components/dropdowns/#options
